src/Symfony/Component/DependencyInjection/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ CHANGELOG
  * Allow #[When] to be extended
  * Change the signature of `ContainerAwareInterface::setContainer()` to `setContainer(?ContainerInterface)`
  * Deprecate calling `ContainerAwareTrait::setContainer()` without arguments
+ * Deprecate using numeric parameter names
```

Comment:
Ah, can you also update `UPGRADE-6.2.md`?

Reply:
Done :).